url/query/operators.go

```diff
@@ -0,0 +1,16 @@
+package query
+
+// basic operators
+const (
+	// sorting
+	ASC  = "asc"  // ascending order
+	DESC = "desc" //descending order
+	IN   = "in"   // in
+	NIN  = "nin"  // not in
+	EQ   = "eq"   // equal
+	NEQ  = "neq"  // not equal
+	GT   = "gt"   // greater than
+	GTE  = "gte"  // greater than or equal
+	LT   = "lt"   //less than
+	LTE  = "lte"  //less than or equal
```

Comment:
Can we please respect the Go naming conventions? This should be `PascalCase`

Reply:
those are acronyms, and (I believe) in Go acronym fields should be uppercased. Will check this to be sure..

Reply:
Ah, yeah, right. But still looks weird to me for `ASC` and `DESC`.

Reply:
ASC and DESC true, could be as 'Asc' and 'Desc'